decidim-accountability/spec/system/admin/admin_imports_projects_to_accountability_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Admin imports projects to accountability", type: :system do
+  let(:manifest_name) { "accountability" }
+  let(:resource_controller) { Decidim::Accountability::Admin::ImportResultsController }
+  let!(:user) { create(:user, :admin, :confirmed, organization: organization) }
+  let!(:organization) { create(:organization) }
+  let!(:participatory_space) { create(:participatory_process, organization: organization) }
+  let(:accountability_component_published_at) { nil }
+
+  include_context "when managing a component as an admin"
+
+  before do
+    switch_to_host(organization.host)
+    login_as user
+    visit_component_admin
+  end
+
+  context "when there are no budgets components to import" do
+    before do
+      click_on "Import projects to results"
+    end
+
+    it "shows no component to select" do
+      expect(page).to have_content t("projects_import.new.no_components", scope: "decidim.accountability.admin")
```

Comment:
Generally we shouldn't rely on the translations within the system specs. Just write the text as the user is expecting to see there. The idea of the system specs is to document a use case on how we'd assume the user to be using the service.

Please go through also similar cases within the other tests.